src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/Wizard.vue

```diff
@@ -16,7 +16,39 @@
     <v-stepper-window v-model="wizardStore.step">
       <v-stepper-window-item v-for="(step, index) in Object.values(wizard.steps)" :key="step.stage" :value="index + 1">
         <v-container>
-          <h3>{{ step.title }}</h3>
+          <v-row class="justify-space-between">
```

Comment:
Will this go against what @SoLetsDev has been working on in references flow? We are trying to white-label the Wizard component as much as possible and put application specific logic in the Application component, could you use a slot for this dialog? https://vuejs.org/guide/components/slots.html#named-slots

Thinking `<dialog></dialog>`, and pass a PrintConfirmationDialog component — would that work here?

Reply:
hmm good point. Though it might be a Print Confirmation dialog that's specific to the review screen? 

I think this should be abstracted out as well. Just the header section perhaps. 

ex. 

<slot name=stepper-window-item-header>
</slot>

then in your applications.vue 

you can add in 
<template #stepper-window-item-header>
that review logic can be baked in here.
</template>

Reply:
Implemented the changes.